Fixed #28 and #53

vzhovtyuk opened the pull request.

lippisch commented on de.jwic.controls/src/de/jwic/controls/tableviewer/DefaultTableRenderer.java:
```diff
@@ -91,6 +91,9 @@ public void renderTable(RenderContext renderContext, TableViewer viewer, TableMo
 				continue;
 			}
+			if (tc.getWidth() == 0) {
+				tc.setWidth(100);
```
I know that this was done before also, but I think it is not a good behavior to change the "configuration" of the table during rendering. I'd rather suggest to interpret a 0 as 100. Something like below....
 ```java
     int width = tc.getWidth() == 0 ? DEFAULT_COLUMN_WIDTH : tc.getWidth();
```

vzhovtyuk commented:
Comment was addressed.
The summary of last changes:
- extracted separate method for calculate effective width, updated table, header and footer with method call
- table column configuration is not changed
- removed 0 column width checks since effective width always more than 0
- added 0px column width case to jwic samples
